rosbridge_server/test/websocket/smoke.test.py

```diff
@@ -1,215 +1,101 @@
 #!/usr/bin/env python
-import json
+import os
 import sys
 import unittest
 
-import launch
-import launch.actions
-import launch_ros
-import launch_ros.actions
 import rclpy
 import rclpy.task
-from autobahn.twisted.websocket import WebSocketClientFactory, WebSocketClientProtocol
-from rcl_interfaces.srv import GetParameters
-from rclpy.executors import SingleThreadedExecutor
 from std_msgs.msg import String
-from twisted.internet import reactor
-from twisted.internet.endpoints import TCP4ClientEndpoint
 from twisted.python import log
 
-log.startLogging(sys.stderr)
+sys.path.append(os.path.dirname(__file__))  # enable importing from common.py in this directory
```

Comment:
There might be a better way to make this import work but I wasn't aware of one. For comparison, the rosbag2 tests use `APPEND_ENV PYTHONPATH=${CMAKE_CURRENT_SOURCE_DIR}` in their CMake rules, but we're using `add_launch_test()` in this package which doesn't seem to support the same flags. cc @emersonknapp 

https://github.com/ros2/rosbag2/blob/ccc0a9ccd7991348f5b91b29f7f267336f17016f/rosbag2_py/CMakeLists.txt#L163-L168